profiling/pgstac/profile-pgstac.py

```diff
@@ -54,34 +137,54 @@ def tile(
             )
             search_info = cursor.fetchone()
             mosaic_id = search_info.id
-    
+
     backend = PGSTACBackend(pool=pool, input=mosaic_id)
-    
+
     def assets_for_tile(x: int, y: int, z: int) -> List[Dict]:
         """Retrieve assets for tile."""
         bbox = backend.tms.bounds(morecantile.Tile(x, y, z))
-        return backend.get_assets(Polygon.from_bounds(*bbox))    
-    
+        return backend.get_assets(Polygon.from_bounds(*bbox))
+
     """Get Tile from multiple observation."""
-    mosaic_assets = assets_for_tile(
-        tile_x,
-        tile_y,
-        tile_z,
-    )
+
+    # PGSTAC Timing
+    with Timer() as t:
+        mosaic_assets = assets_for_tile(
+            tile_x,
+            tile_y,
+            tile_z,
+        )
+        find_assets = t.from_start
```

Comment:
`from_start` is missing from my profiler.main.Timer class - did you add it?

Reply:
ohh maybe not 🤦

Reply:
we should add it https://github.com/developmentseed/titiler/blob/014aea22c8d91d65f9e1b21450a9916c32dc41b1/titiler/utils.py#L95-L98

Reply:
I was able to resolve some attribute missing errors by using t.elapsed and moving the call out of the block, but not sure that was what was intended.

Reply:
`t.elapsed` is fine!

Reply:
I'll let you merge if you want 🙏